modules/fscloud/variables.tf

```diff
@@ -16,6 +16,12 @@ variable "es_name" {
   type        = string
 }
 
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event streams will be created"
```

Comment:
Any reason why this is different from solutions/standard/variables.tf?

>   description = "The region where the IBM Event Streams for IBM Cloud are created and all resources provisioned by this solution are located."

Reply:
I've made the variable description consistent.